src/test/scala-2.11/markets/participants/TestLiquidityDemander.scala

```diff
@@ -45,6 +41,10 @@ case class TestLiquidityDemander(initialDelay: FiniteDuration,
       context.system.scheduler.scheduleOnce(initialDelay, self, SubmitMarketBidOrder)
   }
 
+  val outstandingOrders = mutable.Set.empty[Order]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2093629412/issues/source?bid=2921361&fileBranchId=3140879#l44)